assets/helpers/visitToken/actions.js

```diff
@@ -0,0 +1,15 @@
+// @flow
+
+// ----- Types ----- //
+
+export type Action = { type: 'SET_VISIT_TOKEN', name: string };
+
+
+// ----- Actions Creators ----- //
+
+export function setToken(name: string): Action {
```

Comment:
Conventionally this [action creator](https://redux.js.org/basics/actions#action-creators) would take the same name as the action; in this case `setVisitToken`.